examples/ibm-certificate-manager/README.md

```diff
@@ -77,6 +77,15 @@ data "ibm_certificate_manager_certificates" "certs"{
     certificate_manager_instance_id=data.ibm_resource_instance.cm.id
 }
 
+```
+`Get details of certificate:`
+
+```hcl
+
+data "ibm_certificate_manager_certificate_details" "certificate"{
```

Comment:
ibm_certificate_manager_certificate_details to bm_certificate_manager_certificate